tests/models/nn/transformers/test_bert4rec.py

```diff
@@ -114,6 +123,38 @@ def get_trainer() -> Trainer:
 
         return get_trainer
 
+    @pytest.fixture
+    def get_custom_trainer_func(self) -> TrainerCallable:
+        def get_trainer_func(**kwargs: Unpack[KwargsSpec]) -> Trainer:
+            # internal logic for kwargs
+            max_epochs = kwargs["max_epochs"]
+            accelerator = kwargs["accelerator"]
+            return Trainer(
+                max_epochs=max_epochs,
+                min_epochs=2,
+                deterministic=True,
+                accelerator=accelerator,
+                enable_checkpointing=False,
+                devices=1,
+            )
+
+        return get_trainer_func
+
+    @pytest.fixture
+    def get_custom_val_mask_func(self) -> ValMaskCallable:
+        def get_val_mask_func(interactions: pd.DataFrame, **kwargs: InitKwargs) -> np.ndarray:
```

Comment:
```suggestion
        def get_val_mask_func(interactions: pd.DataFrame, val_users: tp.List[int]) -> np.ndarray:
```
(see comment for trainer func)